src/components/splat.js

```diff
@@ -0,0 +1,910 @@
+  /* global Worker, Blob, fetch */
+  import { registerComponent } from '../core/component.js';
+  import THREE from '../lib/three.js';
+
+  registerComponent('splat', {
+    schema: {
+      src: {type: 'asset'},
+      cutoutEntity: {type: 'selector'},
+      pixelRatio: {type: 'number', default: 1},
+      xrPixelRatio: {type: 'number', default: 0.5}
```

Comment:
These two properties don't belong on the component. They should probably be made part of the `renderer` system. Though the pixelRatio is a bit tricky given that we default to `window.devicePixelRatio`, so maybe express it as a factor on top of the `devicePixelRatio`?

Reply:
yeah. don't love the API yet either